src/scroll_list/ScrollList.js

```diff
@@ -423,7 +423,7 @@ define(function(require) {
             padding: 0,
             persistZoom: false,
             scaleLimits: { minimum: 1, maximum: 3 },
-            touchScrollingEnabled: true,
+            mousePanningEnabled: true,
```

Comment:
In 99% of our use cases don't we want to disable mouse panning? If so, maybe we should change the default to false so that we don't force all consumers to disable it.